### ISSUE TITLE
autopep8 problem

### ISSUE THREAD

**Issue by platinhom** [spyder-ide/spyder-notebook#200]
## Description of your problem

**What steps will reproduce the problem?**

1. Install autopep8
2. Open Autopep8 plugins in Jupyter-Notebook
3. Open Spyder

**What is the expected output? What do you see instead?**

I don't see any warning in Jupyter-Notebook, but when I open spyder, a warning window appears:

```
[autopep8] Sorry, can't use kernel language python3.
Configurations are currently only defined for the following languages:
python
See readme for more details.
```

## Versions and main components

* Spyder Version: 3.3.3
* Python Version: 3.6.6
* Qt Version:
* PyQt Version:  PyQT5
* Jupyter Notebook Version: 5.7.6
* Operating system: MacOS 10.11

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

```
IPython >=4.0     :  7.3.0 (OK)
cython >=0.21     :  None (NOK)
jedi >=0.9.0      :  0.13.3 (OK)
matplotlib >=2.0.0:  3.0.3 (OK)
nbconvert >=4.0   :  5.4.1 (OK)
numpy >=1.7       :  1.16.2 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
psutil >=0.3      :  5.6.1 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pyflakes >=0.6.0  :  2.1.1 (OK)
pygments >=2.0    :  2.3.1 (OK)
pylint >=0.25     :  2.3.1 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
rope >=0.9.4      :  0.12.0 (OK)
sphinx >=0.6.6    :  1.8.5 (OK)
sympy >=0.7.3     :  1.3 (OK)
```

**Comment by platinhom** [comment]
I solved this problem by editing the json prettifying of autopep8 configuration in Jupyter-Notebook nbextension: 

```json
{
    "python": {
        "library": "import json\nimport autopep8",
        "prefix": "print(json.dumps(autopep8.fix_code(u",
        "postfix": ")))"
    },
    "python3": {
        "library": "import json\nimport autopep8",
        "prefix": "print(json.dumps(autopep8.fix_code(u",
        "postfix": ")))"
    }
}
```

**Comment by bertrand-faure** [comment]
Hi platinhom,

I'm running into the same issue. Could you be a bit more specific regarding the location of the file you have edited? I've found a relevant entry in the site-packages folder jupyter_contrib_nbextensions\nbextensions\code_prettify\autopepe8.js, but the warning window still pops up. Thanks!

**Comment by platinhom** [comment]
@bertrand-faure Hi. You should edit the contents in the jupyter-notebook extensions as :

![autopep8 at nbextentsions](https://user-images.githubusercontent.com/3596077/56327066-3c457e80-61ab-11e9-9b37-d5b28f29811c.png)

Edit contents at the following input box:

![Configuration for Autopep8](https://user-images.githubusercontent.com/3596077/56327106-70b93a80-61ab-11e9-9306-262bbb54756d.png)

**Comment by jitseniesen** [comment]
This is all about version 0.1 of the plugin, we never managed to reproduce it and we have just released version 0.3, so I am closing this issue. If anybody encounters the same problem with version 0.3, please open a new issue so that we can get to the bottom of it.
